### ISSUE TITLE
Play Duration and count suddenly incorrect

### ISSUE THREAD

**Issue by kjames79** [Tautulli/Tautulli#864]
**Version:**
1.4.12

**Branch:**

**Commit hash:**

**Operating system:**
Windows 7

**Python version:**

**What you did?**

**What happened?**
Play Duration and count suddenly incorrect

**What you expected?**

**How can we reproduce your issue?**

**What are your (relevant) settings?**

**Link to logs:**
let me know how you would like to receive these and also I can provide screenshots.
This only started yesterday. I have been using Plexpy without error for 58 days.

**Comment by JonnyWong16** [comment]
Please follow the issues guidelines: https://github.com/JonnyWong16/plexpy#issues

**Comment by novacrazy** [comment]
I'm also encountering this issue with PlexPy 1.4.12 since PMS 1.2.1.2854-3ed5ca1, currently on PMS 1.2.2.2857-d34b464, both running on Ubuntu 16.04 64-bit

![plexpyconfig](https://cloud.githubusercontent.com/assets/4708645/19216675/b666b324-8d8b-11e6-8f9d-ba6175d50f94.PNG)

Here is a screenshot of the latest entry in the history tab:
![sharktale](https://cloud.githubusercontent.com/assets/4708645/19216646/1569c5b6-8d8a-11e6-8d8d-db846ce8e885.PNG)

I really hope I haven't been watching Shark Tale for nearly three days straight. 

Despite that I only watched it for a couple seconds to check quality, and it is no longer playing, and that I've restarted PlexPy and PMS multiple times, it still updates that time constantly. 

It also is not updating it with newer watches, so the history is totally non-functional for me now. 

The log functionality in PlexPy also isn't filtering correctly. I can plainly see a few warnings, yet filtering by warnings says there aren't any results. I'm not comfortable uploading the entire logs here.

**Comment by JonnyWong16** [comment]
@novacrazy 

Go to `http://localhost:8181/delete_temp_sessions` and manually delete that one history entry for Shark Tale.

**Comment by novacrazy** [comment]
Hmm. That seems to have fixed it for now. History is updating again, as well. Odd, but thank you.

**Comment by JonnyWong16** [comment]
Not odd. You had a session stuck inside the database.

**Comment by kjames79** [comment]
I tried what you suggested novacrazy do. that link didn't seem to take me to anything. Here's what my issue looks like

https://drive.google.com/file/d/0BwO2arTS7M49NEU2UC1vTlQ0Wms/view?usp=sharing

https://drive.google.com/file/d/0BwO2arTS7M49NmphSDF1a0ZRcGM/view?usp=sharing

https://drive.google.com/file/d/0BwO2arTS7M49TEE1QWpUQXBhYUE/view?usp=sharing

https://drive.google.com/file/d/0BwO2arTS7M49Y3I2d1YtaGRQb0U/view?usp=sharing

https://drive.google.com/file/d/0BwO2arTS7M49d3RXYmdHZVVyTkk/view?usp=sharing

**Comment by JonnyWong16** [comment]
You have to delete that history entry. History > Delete mode.

**Comment by kjames79** [comment]
noticed those links are dead. anyway let me try again what you're saying to do

**Comment by kjames79** [comment]
got it. thank you!

**Comment by JonnyWong16** [comment]
This should be fixed (prevented from happening again) on v1.4.13.
